nss-day-cohort-29/reactive-nutshell-starrywick
# cw-events1

Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Type of change

[X ] New feature (non-breaking change which adds functionality)
The add and list features work

Testing Instructions for Change Made
git fetch --all
git checkout branch cw -events1
run json server on port 5002
npm start in reactive-nutshell
A list of articles should appear
click on events in navbar
try adding events of your choice
Test edit link to see if form pulls up for you - I created a seperate component EventEdit
here is a sample database json:
 "events": [
    {
      "userId": 2,
      "eventDate": "2019-01-30",
      "name": "Something showed up on DOM Party",
      "location": "NSS",
      "id": 1
    },
    {
      "userId": 2,
      "eventDate": "2019-01-31",
      "name": "Something else showed up on DOM National Holiday",
      "location": "Destin",
      "id": 2
    },
    {
      "userId": 2,
      "eventDate": "2019-02-02",
      "name": "Ground Hog Day",
      "location": "Philadelphia",
      "id": 3
    },
    {
      "userId": 2,
      "eventDate": "2019-02-22",
      "name": "Lab Day",
      "location": "NSS",
      "id": 4
    }
  ],

Checklist:
[X ] My code follows the style guidelines of this project
[X ] I have performed a self-review of my own code
[X ] My changes generate no new warnings
[X ] I have added test instructions that prove my fix is effective or that my feature works
